opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -273,10 +291,13 @@ public SourceResponse query(QueryRequest queryRequest) throws UnsupportedQueryEx
                 Metacard metacard = null;
                 List<Result> resultQueue = new ArrayList<Result>();
                 try {
-                    metacard = inputTransformer.transform(responseStream);
-                } catch (IOException e) {
-                    LOGGER.debug("Problem with transformation.", e);
-                } catch (CatalogTransformerException e) {
+                    final byte[] bytes = IOUtils.toByteArray(responseStream);
```

Comment:
Seems like that kind of magic number should be configurable, unless I'm missing something. Is there a reason it's hardcoded in your example? Otherwise it's only performant for systems with the right balance of available memory, hard drive speed, and response size.